### PR TITLE
 Spotinst - add ocean module

### PR DESCRIPTION
**Pull request by jeffnoehren**
##### SUMMARY
Adding new Spotinst module - Ocean

##### ISSUE TYPE
- New Module Pull Request

##### ADDITIONAL INFORMATION
```paste below
ok: [localhost] => {
    "result": {
        "changed": true, 
        "failed": false, 
        "group_id": "o-35124875", 
        "message": "Created Ocean Cluster successfully."
    }
}
```

### CODE REVIEW

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -324,6 +324,7 @@ lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E322
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E323
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E324
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E326
+lib/ansible/modules/cloud/spotinst/spotinst_ocean_cloud.py E326
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt]: New ignores should not be added. Correct the underlying issue in the module. If the issue is in an existing module_utils, fix that issue in a separate PR first.

**Review by mattclay — DISMISSED** [review]

File: test/sanity/validate-modules/ignore.txt (hunk 1 above)
Comment (jeffnoehren) [reply test/sanity/validate-modules/ignore.txt]: ok this has been taken out!

**Comment by talzur** [comment]
shipit

**Comment by kmcgrath** [comment]
@mattclay Is there anything else you need from our side to get this task in the release flow?

File: changelogs/fragments/spotinst-add-ocean-resource.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - spotinst - Added Ocean resource
```
Comment (mattclay) [inline changelogs/fragments/spotinst-add-ocean-resource.yml:2]: New modules should not have changelog entries.

**Pull request closed without merging**